colaraz/lms/templates/instructor/instructor_dashboard_2/instructor_dashboard_2.html

```diff
@@ -103,18 +104,22 @@
         <main id="main" aria-label="Content" tabindex="-1">
         <section class="instructor-dashboard-content-2" id="instructor-dashboard-content">
           <header class="page-info-header">
+              <%
+                course_overview = CourseOverview.get_from_id(course.id)
+                course_image_url = course_overview.course_image_url
+              %>
               <div class="image-frame">
-                  <img src="${static.url('images/course.jpg')}" alt="Autodesk Maya">
+                <img src="${course_image_url if not '/static/studio/images/pencils.jpg' in course_image_url else static.url('images/course.jpg')}" alt="${course_overview.display_name_with_default}">
```

Comment:
We are checking for `/static/studio/images/pencils.jpg` because we want to use our own default image. Correct? 
If so, wouldn't it be better to do this while saving the course?

Reply:
@imhassantariq has done this for UCSD. He can guide you better

Reply:
I have already discussed it with @imhassantariq. If colaraz asks us to do that, we will definitely do that.

Reply:
You sure? Because they don't know the code base as we do. We tried the same thing for UCSD but decided to change the behavior because we cannot change every HTML with to display this default picture.

Reply:
@mumarkhan999 if we can do better, please do it. At least, you can ask ahsan/babar on the ticket you are working on.